internal/vendors/supermicro/sum.go

```diff
@@ -288,7 +292,7 @@ func (s *sum) ConfigureBIOS() (bool, error) {
 // EnsureBootOrder ensures BIOS boot order so that boot from the given allocated OS image is attempted before PXE boot.
 func (s *sum) EnsureBootOrder(bootloaderID string) error {
 	s.bootloaderID = bootloaderID
-	if s.boardModel == X13DDW_A {
+	if s.boardModel == X13DDW_A || s.boardModel == X12DPT_B6 {
 		s.log.Infow("GPU board detected, skip bios modification", "board", s.boardName)
```

Comment:
Maybe a more generic error message since X12 is not a GPU board for us? "Broken board..." or something?

Reply:
Right, will fix